nutanix/provider.go

```diff
@@ -152,6 +152,7 @@ func Provider() *schema.Provider {
 			"nutanix_recovery_plan":           resourceNutanixRecoveryPlan(),
 			"nutanix_service_group":           resourceNutanixServiceGroup(),
 			"nutanix_address_group":           resourceNutanixAddressGroup(),
+			"nutanix_foundation_image_nodes":  ResourceFoundationImageNodes(),
```

Comment:
```suggestion
			"nutanix_foundation_image_nodes":  resourceFoundationImageNodes(),
```
@siddharth-nutanix we keep every resource func private right ?

Reply:
yes. I missed it .